bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
`os.system` works fine in an empty environment?

```
$ env -i python -c 'import os; os.system("echo hello")'
hello
```

or do you have something more specific in mind?

even `os.path.expanduser` works fine without having `$HOME` defined:

```
$ env -i python -c 'import os; print os.path.expanduser("~"); print os.getenv("HOME", "<home_not_set>")'
/Users/kehoste
<home_not_set>
$ python -c 'import os; print os.path.expanduser("~"); print os.getenv("HOME", "<home_not_set>")'
/Users/kehoste
/Users/kehoste
```